src/main.css

```diff
@@ -1,17 +1,37 @@
-* {
-    margin: 0;
-    padding: 0;
-}
-
 /* global variables (custom properties) */
 :root {
     --primary-color: #099dda;
-    --bg-color: #212f3d;
-    --color: white;
-    --input-bg-color: #354453;
     --font-family: "Courier New", Courier, monospace;
 }
-body {
+
+.light-theme {
+    --primary-bg-color: white;
+    --secondary-bg-color: #F8F8F8;
```

Comment:
on my display is the colorstep( Hue) between white an  #F8F8F8 to small. The background for the autocompletion-field is visible but the background in the toggle-button for the UI-Theme is not visible (on my screen). I would prever #e8e8e8, which is still noticeable

Reply:
thx, i will  borrow that. 

The main focus of this PR is not to finish the layout, but to provide a basic infrastructure, upon which
our specialist can do his work (-;